Instagram/Cells/StoriesViewCell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  StoriesViewCell.swift
+//  Instagram
+//
+//  Created by Aleksandr Dorofeev on 19.10.2022.
+//
+
+import UIKit
+
+/// StoryCell configuration.
```

Comment:
```suggestion
/// Ячейка истории
```
Давай нормальные названия